python/foglamp/services/core/api/plugins/install.py

```diff
@@ -246,20 +246,29 @@ def copy_file_install_requirement(dir_files: list, plugin_type: str, file_name:
 
 
 def install_package_from_repo(name: str, pkg_mgt: str, version: str) -> tuple:
-    stdout_file_path = "/data/plugins/output.txt"
-    cmd = "sudo {} -y install {}".format(pkg_mgt, name)
-    if version:
-        cmd = "sudo {} -y install {}={}".format(pkg_mgt, name, version)
-
-    ret_code = os.system(cmd + " > {} 2>&1".format(_FOGLAMP_ROOT + stdout_file_path))
     msg = ""
-    with open("{}".format(_FOGLAMP_ROOT + stdout_file_path), 'r') as fh:
+    plugin_dir = '/plugins/'
+    stdout_file_name = "output.txt"
+    stdout_file_path = "/{}/{}".format(_PATH, stdout_file_name)
+
+    if not os.path.exists(_PATH):
+        os.makedirs(_PATH)
+
+    cmd = "sudo {} update > {} 2>&1".format(pkg_mgt, stdout_file_path)
```

Comment:
`update` command is not needed here as we are calling [fetch_available_package](https://github.com/foglamp/FogLAMP/pull/1627/files#diff-fdf85c87517aa764a4ae1b7bf9e8e427R75) before plugin install to ensure that requested plugin is not available for the given repository.